modules/ipi-install-bmc-addressing-for-dell-idrac.adoc

```diff
@@ -17,10 +17,11 @@ platform:
         role: <master | worker>
         bmc:
           address: <address> <1>
-          username: <user>
+          username: <user> <2>
           password: <password>
 ----
 <1> The `address` configuration setting specifies the protocol.
+<2> The `username` configuration setting must specify a user with `Administrator` privileges.
```

Comment:
Same

Reply:
Same answer. I have iDRAC 7, and I looked online for iDRAC9, and it's the same too.